recipes/proxy/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This is a Dockerfile for use with the ddev-webserver image
 # It should be in .ddev/web-build/Dockerfile
 # and then you can expand on it.
-ARG BASE_IMAGE=drud/ddev-webserver:v1.9.1
+ARG BASE_IMAGE=ddev/ddev-webserver:v1.9.1
```

Comment:
This isn't correct, the dockerhub repo will not yet be updated to use drud. Hopefully we can do that later. Not sure we have control of it.

Reply:
reverted.